src/agent/oneplatform-controller.ts

```diff
@@ -0,0 +1,328 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*!
+ * @module debug/oneplatform-controller
+ */
+
+import {ServiceObject} from '@google-cloud/common';
+import * as assert from 'assert';
+import * as qs from 'querystring';
+import * as t from 'teeny-request';
+
+import {URL} from 'url';
+
+import {Logger, ResolvedDebugAgentConfig} from './config';
+import {Controller} from './controller';
+import {StatusMessage} from '../client/stackdriver/status-message';
+import {Debug} from '../client/stackdriver/debug';
+import {Debuggee} from '../debuggee';
+import * as stackdriver from '../types/stackdriver';
+
+const BREAKPOINT_ACTION_MESSAGE =
+  'The only currently supported breakpoint actions' + ' are CAPTURE and LOG.';
```

Comment:
nit: I don't think two strings need to be appended together here, confused me slightly.

Reply:
Cute.  I'll fix that later; I assume that the linter moved the linebreak and it's solidly nonsense now.